cms/tests/test_app_registration.py

```diff
@@ -53,4 +53,186 @@ def test_imports_cms_apps_files(self):
     ])
     def test_raises_exception_raised_in_cms_file(self):
```

Comment:
Should be a new line here. Picky I know. :-)

Reply:
Being picky back :wink: AFAIK pep8 doesn't require it and I actually prefer it visually without. However, if there is a general convention in django-cms to add an empty line (I haven't noticed this, but feel free to direct me to examples) then I can keep to it for consistency.